admin/src/main/scala/io/buoyant/admin/AdminConfig.scala

```diff
@@ -2,4 +2,4 @@ package io.buoyant.admin
 
 import io.buoyant.config.types.Port
 
-case class AdminConfig(port: Port = Port(9990))
+case class AdminConfig(port: Port = Port(9990), shutdownGraceMs: Int = 10000)
```

Comment:
I would make `shutdownGraceMs` optional by making it an `Option[Int]`.  In `Linkerd.scala` we can apply a default value and convert to a `Duration`.